soh/soh/SohMenuBar.cpp

```diff
@@ -1537,6 +1537,10 @@ void DrawRandomizerMenu() {
                 "Play unique fanfares when obtaining quest items "
                 "(medallions/stones/songs). Note that these fanfares are longer than usual."
             );
+            UIWidgets::PaddedEnhancementCheckbox("Freestanding Mystery Items", "gFreestandingMystery", true, false);
+            UIWidgets::Tooltip(
+                "Displays a \"Mystery Item\" model in place of any freestanding/GS/shop items that were shuffled,\n"
```

Comment:
nit: Don't need to manually specify line breaks just for long lines

Reply:
Oh, right, I remembered seeing that tooltip in practice and it didn't look like it even did anything XD